source/utilities.tex

```diff
@@ -7607,8 +7616,9 @@
 \effects
 Equivalent to:
 \begin{codeblock}
-return ::new(static_cast<void*>(p))
-  T(make_obj_using_allocator<T>(alloc, std::forward<Args>(args)...));
+return apply([&](auto&&...xs) {
+         return construct_at(p, std::forward<decltype(xs)>(xs)...);
```

Comment:
Can't we editorially use `[&]<class... U>(U&&.. xs) {` and `std::forward<U>(xs)...` here?

Reply:
I made this change in a separate edit - will leave it up to @zygoloid to see if he wants to include it as part of the motion or not.